docs/How-to-setup-mysql-plugin.md

```diff
@@ -0,0 +1,14 @@
+# How to setup Mysql plugin for OMS agent
+
+How to enable mySQL plugin to collect mySQL server stats and perf data
+* Machine has to have libmysqlclient.so installed (which can be downloaded and installed through mysql-devel.
+					yum install mysql-devel or apt-get mysql-devel for Debian systems)
```

Comment:
Your instructions are not correct for Debian/Ubuntu, please verify proper command and update.